library/core/src/marker.rs

```diff
@@ -984,10 +999,18 @@ pub trait PointerLike {}
 #[cfg_attr(not(bootstrap), lang = "const_param_ty")]
 #[unstable(feature = "adt_const_params", issue = "95174")]
 #[rustc_on_unimplemented(message = "`{Self}` can't be used as a const parameter type")]
-pub trait ConstParamTy: StructuralEq {}
+#[allow(multiple_supertrait_upcastable)]
+pub trait ConstParamTy: StructuralEq + StructuralPartialEq {}
+
+/// Derive macro generating an impl of the trait `Copy`.
```

Comment:
Did you mean `ConstParamTy` instead of `Copy` here?

Reply:
I did! The PR with that was already merged though, feel free to send in a fix.